.gitignore

```diff
@@ -104,3 +104,6 @@ cataclysm.a
 
 # coredumps
 *.core
+
+# sublime-text workspace
+*.sublime-workspace
```

Comment:
I don't think you'll want to add the sublime workspaces for the whole project to .gitignore, they will probably need to be updated in the future, I know the codeblocks project file gets modified frequently. If you had to update the workspace to make the project work, include those changes for review. 
Iirc several libraries were added to the codeblocks project not to long ago, I don't think the sublime workspace was updated at the same time.